vividus-http-client/src/main/java/org/vividus/http/client/HttpClient.java

```diff
@@ -161,7 +162,8 @@ private static URI getRequestUri(ClassicHttpRequest request) throws IOException
     private static void configureBasicAuth(boolean usePreemptiveBasicAuthIfAvailable, String plainUserInfo,
             HttpHost host, HttpClientContext internalContext)
     {
-        UserInfo userInfo = UriUtils.parseUserInfo(plainUserInfo);
+        String plainUserInfoDecoded = URLDecoder.decode(plainUserInfo, StandardCharsets.UTF_8);
+        UserInfo userInfo = UriUtils.parseUserInfo(plainUserInfoDecoded);
```

Comment:
would you mind adding integration test for this case? you can use https://httpbin.org/#/Auth